src/Game/Player.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react"
+
+export default function Player({id, name, symbol, updatePlayerTitle}) {
+
+    const [isEditing, setIsEditing] = useState(false)
+
+    const handleEdit = () => {
+        if (name === '') {
+            alert('Name cannot be empty');
+        } else {
+            setIsEditing((editing) => !editing);
+        }
+    }
+
+    const handlePlayerNameChange = (value) => {
+        updatePlayerTitle(id, value)
+    }
+
+    let playerName = <span className="player-name">{name}</span>
+    if(isEditing) {
+        playerName = <input className="player-name" value={name} onChange={ (e) => handlePlayerNameChange(e.target.value) }></input>
+    }
+
+    return (
+        <li>
+            <span className="player">
+                {playerName}
+                <span className="player-symbol">{symbol}</span>
+            </span>
+            <button onClick={handleEdit}> {isEditing ? 'Save' : 'Edit'} </button>
+        </li>
+    )
+}
```

Comment:
The font color could be changed for better visibility

Reply:
I've changed the appearance of button. It is more visible now.